### DISCUSSION TITLE
Load `gitfast` from OMZP without `zi ice svn`

### DISCUSSION THREAD

**Discussion by gabyx** [z-shell/zi#283] [Q&A] [answered]
Is it possible to load the `gitfast` plugin from [`OMZP:gitfast`](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/gitfast) without the use of `zi ice svn` because I dont want to install SVN.

I tried to use
```shell
zi ice multisrc='git-prompt.sh gitfast.plugin.zsh'
zi snippet OMZP::gitfast
```

but that does not work, I suppose the gitfast needs to download two files `git-prompt.sh` as well. So how do I achieve this?

**Comment by ss-o** [comment, marked as answer]
Hi @gabyx :wave:

I'm sorry for not getting back to you sooner. I am unaware of the reliable way to get specific folders from the git repository ([related topic](https://stackoverflow.com/questions/7106012/download-a-single-folder-or-directory-from-a-github-repo)). If not subversion, then the custom script/snippet for custom tasks can be used.

For such scenarios, I recommend creating your zsh-plugin which will set it up for you. It is relatively easy to achieve, easy to use and can be loaded/unloaded when required. :sparkles: 

  The zsh-plugin can be a private/public repository and installed as:

```shell
zi load gabyx/git-ops
```

If you are interested I can provide the required steps for creating a zsh-plugin which will solve your issue. :dart:
